server/app/views/applicant/ApplicantProgramBlockEditView.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
If we're going to call the java classes `View`s I think we should call these something else, like `ApplicantProgramBlockEditTemplate.html` to help disambiguate.